appveyor.yml

```diff
@@ -86,15 +88,21 @@ before_build:
 build_script: c:\projects\wxwidgets\build\tools\appveyor.bat
 
 before_test:
+- cmd: |
+    echo Getting and launching httpbin.
+    rmdir %GOROOT% /s /q
+    mkdir %GOROOT%
+    appveyor DownloadFile https://golang.org/dl/go%GOVERSION%.windows-amd64.msi
```

Comment:
Just to be clear: we can't use any of [Go versions](https://www.appveyor.com/docs/windows-images-software/#golang) already available on Appveyor?